src/style.css

```diff
@@ -1 +1,85 @@
-/* Styles go here */
+@font-face {
+  font-family: Roboto, sans-serif;
+  src: url(https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap);
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+  cursor: pointer;
+}
+
+.header {
+  /* display: flex;
+  position: sticky; */
+  top: 0;
+  text-transform: uppercase;
+  font-weight: 500;
+  background: #fff;
```

Comment:
it's best to set font-weight in the html

also you had the correct idea about `display: flex` here. your header has two items and you want them to be positioned on the opposite sides. you need `justify-content: space-between` for that.

also no need for positioning (sticky and top)